src/controllers/credential-status.ts

```diff
@@ -873,13 +874,17 @@ export class CredentialStatusController {
 						symmetricKey: '',
 					},
 				} as ITrackOperation;
-				const trackResult = await identityServiceStrategySetup.agent.trackOperation(trackResourceInfo);
+				const trackResult = await identityServiceStrategySetup.agent
+					.trackOperation(trackResourceInfo)
+					.catch((error) => {
+						return { error };
+					});
 				if (trackResult.error) {
-					return response.status(StatusCodes.INTERNAL_SERVER_ERROR).json({
-						updated: false,
-						error: trackResult.error,
-					} as UpdateUnencryptedStatusListUnsuccessfulResponseBody);
+					console.error(`Tracking Error: ${trackResult.error}`);
 				}
+				return response.status(StatusCodes.OK).json({
```

Comment:
I think it's not needed, cause in case of success  next line will be called and make the same action